app/src/main/java/fr/free/nrw/commons/feedback/FeedbackDialog.java

```diff
@@ -0,0 +1,80 @@
+package fr.free.nrw.commons.feedback;
+
+import android.app.Dialog;
+import android.content.Context;
+import android.os.Bundle;
+import android.widget.Button;
+import android.widget.CheckBox;
+import butterknife.BindView;
+import butterknife.ButterKnife;
+import butterknife.OnClick;
+import fr.free.nrw.commons.R;
+import fr.free.nrw.commons.feedback.model.Feedback;
+import fr.free.nrw.commons.ui.PasteSensitiveTextInputEditText;
+import fr.free.nrw.commons.utils.ConfigUtils;
+import fr.free.nrw.commons.utils.DeviceInfoUtil;
+import javax.inject.Inject;
+
+/**
+ * Feedback dialog that asks user for message and
+ * other device specifications
+ */
+public class FeedbackDialog extends Dialog {
+
+    @BindView(R.id.btn_submit_feedback)
```

Comment:
We are moving away from Butterknife, see https://github.com/commons-app/apps-android-commons/issues/4664

Reply:
@madhurgupta10 I have removed but butterknife. thanks for feedback